src/main/scala/mesosphere/marathon/api/akkahttp/Rejections.scala

```diff
@@ -19,6 +19,9 @@ object Rejections {
     def noApp(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
       EntityNotFound(Message(s"App '$id' does not exist" + version.fold("")(v => s" in version $v")))
     }
+    def noPod(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
+      EntityNotFound(Message(s"Pod '$id' does not exist" + version.fold("")(v => s" in version $v")))
```

Comment:
Checkout https://github.com/mesosphere/marathon/pull/5754/files#diff-ac498c575af88454a7f4eabdb3498b23. We probably should agree on one pattern. Which do you like more?

Reply:
I'll use yours :)